server/db/index.js

```diff
@@ -1,9 +1,27 @@
 'use strict';
 var db = require('./_db');
-module.exports = db;
+var Sequelize = require('sequelize');
```

Comment:
No need to require `sequelize` into this file given that you're not using it here.